File: examples/cmp/java/Gen.java
import java.util.Random;
import java.io.PrintWriter;

public class Gen {
    public static void main(String[] args) {
        Random rand = new Random();
        PrintWriter out = new PrintWriter(System.out);
        
        final int N  = 1000;
        final int mn = -100000;
        final int mx = 100000;

        int n = rand.nextInt(N);

        out.println(n);

        for(int i = 0; i < n; i++) {
            if(i > 0)
                out.print(" ");
            int ai = (int)Math.floor(Math.random()*(mx-mn+1)+mn);
            out.print(ai);
        }
        out.flush();
    }
}

File: examples/cmp/kotlin/gen.kt
import kotlin.random.Random
fun random(random: Random, from: Int, to: Int): Int {
    return (random.nextInt(from, to)).toInt()
}
fun main(args: Array<String>) {
    var seed = args[1].toInt();
    var rand = Random(seed)
    val out = StringBuilder()
    var N  = 1000
    var Ai = 100000 
    var n = random(rand, 1, N)
    out.appendLine(n)
    for (i in 0..n-1) {
        if(i > 0) {
            out.append(" ")
        }
        out.append(random(rand, -Ai, Ai))
    }
    println(out)
}